v3/internal/commands/build_assets/config.yml

```
@@ -0,0 +1,32 @@
+#    description: Wails Application File
+#    iconName: wailsFileIcon
+#    role: Editor
+#  - ext: jpg
+#    name: JPEG
+#    description: Image File
+#    iconName: jpegFileIcon
+#    role: Editor
+
+# Other data
+other:
+  - name: My Other Data
```

yamllint

[error] no new line character at the end of file

(new-line-at-end-of-file)